tests/queries/tests.py

```diff
@@ -83,6 +85,99 @@ def setUpTestData(cls):
         Cover.objects.create(title="first", item=i4)
         Cover.objects.create(title="second", item=cls.i2)
 
+    @skipIfDBFeature('supports_explaining_query_execution')
```

Comment:
Could you please put these in a new `test_explain.py` file? This file is already a bit too long and unorganized.